yarnitor/static/js/main.js

```diff
@@ -196,6 +225,8 @@ $(document).ready(function() {
     $.fn.dataTable.ext.errMode = 'throw';
     // Immediately try to fetch datetime of last data refresh
     reloadDatetime();
+    // Set the initial initial filter as specified by the URL hash
```

Comment:
What does setting the initial filter mean? Also did you double up "initial" here?

Reply:
Clarified the comment. The code here pulls from the URL hash value `http://foo.com/#something` and sets the filter text box to the value `something`.  This makes it possible to share a filtered view of rows by sharing the yarnitor URL. When othe users visit that URL, they get the same view as you (e.g., "look at these jobs").